From 44305175df3981c0745929ddbe15d0043dcc1e59 Mon Sep 17 00:00:00 2001
From: Peter Harliman Liem <peter.harliman.liem@intel.com>
Date: Mon, 30 Jul 2018 11:08:22 +0800
Subject: [PATCH] Add deprecated info in ltq_system_reset Kconfig

LTQ_SYSTEM_RESET is a legacy reset mechanism from GRX model, kept
only for legacy purpose to those that still require its API.
Users should no longer use this and switch to standard linux reset
interface (with INTEL_SYSCON_RESET).
---
 drivers/char/Kconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/char/Kconfig b/drivers/char/Kconfig
index a0fe7f3646f2..9f806e4d8989 100644
--- a/drivers/char/Kconfig
+++ b/drivers/char/Kconfig
@@ -617,10 +617,12 @@ config LTQ_ICC
 	  Turn on this option to enable icc driver.
 
 config LTQ_SYSTEM_RESET
-	tristate "Lantiq system reset driver"
+	tristate "Lantiq system reset driver (DEPRECATED)"
 	depends on LTQ_ICC
 	default y
 	---help---
 	  Turn on this option to enable system reset driver.
+	  This driver is deprecated. Use standard linux reset
+	  interface with INTEL_SYSCON_RESET instead.
 endmenu
 
